source/codegen/service_helpers.py

```diff
@@ -422,6 +422,24 @@ class CrossDriverSessionDependency(NamedTuple):
     local_name: str
 
 
+def get_driver_shared_resource_repository_ptr_deps(
+    driver_config: dict,
+) -> List[CrossDriverSessionDependency]:
+    resource_repository_deps = [
+        _create_cross_driver_session_dependency(resource_handle_type)
+        for resource_handle_type in get_resource_handle_types(driver_config)
+    ]
+
+    resource_repository_deps[0] = CrossDriverSessionDependency(
```

Comment:
note: I couldn't find a better way to customize field names and alias of "default" repository.